src/spatialdata/_io/format.py

```diff
@@ -45,7 +47,24 @@ def _parse_version(group: zarr.Group, expect_attrs_key: bool) -> str | None:
 
 
 class SpatialDataFormat(CurrentFormat):
-    pass
+    @property
+    def version(self) -> str:
+        return "0.6-dev"
```

Comment:
cc @bogovicj in case defining a specific pre-release version (i.e. dev1 or beta1) would make detecting differences in these upcoming versions more reliable.

See also [ongoing conversation around 0.5.2](https://github.com/ome/ngff/pull/293)

Reply:
Actually, going back and looking at the diff output in the description, I'd suggest we *not* use 0.6 unless this also implements the Zarr v3 work of 0.5.

Reply:
This is a good point. Any suggestion on the string to use? Maybe something more verbose like `0.4-dev-transformations`?

Reply:
Or `0.4-dev-spatialdata	`.

Reply:
* :+1: for starting with `0.4-`
 * :+1: for having "spatialdata" or similar in it
 * you might think about whether or not you have a version field from SD itself
 * I defer to you if you want "dev" in it or not (I don't think the validator, e.g., has special behavior for that. @will-moore?)

Reply:
No, the validator currently just uses the version string to construct URLs for loading the schemas. So it'll need a little patching to parse the "0.4" out of a longer string if it's going to validate those.

Reply:
Thanks @joshmoore. 
> you might think about whether or not you have a version field from SD itself
The SpatialData is versioned here: https://github.com/scverse/spatialdata/blob/a7e9e991c31474c7495b17a5eb8544d7d2eba108/src/spatialdata/_io/format.py#L233

@will-moore I think your "no" answers to "(I don't think the validator, e.g., has special behavior for that. @will-moore?)". Do you have any preference for the `0.4-` string?

Reply:
I can handle `0.4-*` anything is fine.

I tried looking at some spatialdata data in the validator with...

```
$ ome_zarr view mouse_liver_data.zarr
```

 ...looking at the `/labels/segmentation_mask/` the validator is complaining that `/multiscales/0/coordinateTransformations/0` doesn't have `scales` or `translation` objects.
Instead you have `"input":{"axes":[...]}, "output": {"axes":[...]}, "type": "identity"}` which is heading towards the CoordinateSystems RFC-5 https://ngff.openmicroscopy.org/rfc/5/index.html but not quite the same.

Reply:
Yes, I will adjust the coordinate transformations metadata together with @melonora in `ome-zarr-models-py`.

Reply:
If you agree, I'd go then with `0.4-dev-spatialdata` and merge this PR.

Reply:
Thanks Will, but as Josh said here:
> Except this version explicitly does not validate, so perhaps it's enough that it's just not the known "0.4"?
Not passing the validation is expected because the coordinate transformation specification that we implemented was based on an earlier draft and our plan is to finalize it in `ome-zarr-models-py`.

In this PR I made a patch to `ome-zarr-py` to be able to read the tensors even with partially incorrect metadata. We indeed handle the read/write of metadata in `spatialdata`, and use `ome-zarr-py` to read the tensors.

What would be needed here is to agree on an appropriate version number, such as `0.4-dev-spatialdata`, that doesn't imply that validation is passed.

Reply:
Please in this comment you can see the full context of why we want to fix the version string now and we defer passing the validation later: https://github.com/scverse/spatialdata/issues/805#issuecomment-2531706080.